api/Api/Runtime/ApiProvider.cs

```diff
@@ -50,5 +52,16 @@ public static List<Component> GetComponents(uint entity)
 		}
 
 		public static Component? AddComponent(Type t, uint entity) => Instance?.AddComponent(t, entity);
+
+		public static T? CreateUnityType<T>(params object[] args) where T : class => Instance?.CreateUnityType<T>(args);
```

Comment:
We shouldn't expose any Unity code in the API so users won't have to install Unity to make modules.

Reply:
ApiProvider is going to be available to the modules?

Reply:
Yes, it's a part of the API